test/entrypoint/srv_ddl.lua

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env tarantool
+
+require('strict').on()
+_G.is_initialized = function() return false end
+
+local log = require('log')
+local errors = require('errors')
+local cartridge = require('cartridge')
+local ddl = require('ddl')
+
+package.preload['customers-storage'] = function()
+    return {
+        role_name = 'customers-storage',
+        init = function()
+            local engine = os.getenv('ENGINE') or 'memtx'
+            local schema = {
+                spaces = {
+                    customers = {
+                        engine = engine,
+                        is_local = true,
+                        temporary = false,
+                        format = {
+                            {name = 'id', is_nullable = false, type = 'unsigned'},
+                            {name = 'bucket_id', is_nullable = false, type = 'unsigned'},
+                            {name = 'name', is_nullable = false, type = 'string'},
+                            {name = 'age', is_nullable = false, type = 'number'},
+                        },
+                        indexes = {{
+                            name = 'id',
+                            type = 'TREE',
+                            unique = true,
+                            parts = {
+                                {path = 'id', is_nullable = false, type = 'unsigned'},
+                                {path = 'name', is_nullable = false, type = 'string'},
+                            },
+                        }, {
+                            name = 'bucket_id',
+                            type = 'TREE',
+                            unique = false,
+                            parts = {
+                                {path = 'bucket_id', is_nullable = false, type = 'unsigned'},
+                            }
+                        }},
+                        sharding_key = {'name'},
```

Comment:
We need a test that check that select({'=', 'name', {<name>}}) won't lead to map-reduce.

Reply:
decided to postpone it https://github.com/tarantool/crud/issues/213

Reply:
test added, but works on space with additional index by field name used in SELECT's condition
```
    ddl_sharding_key[memtx].test_select_wont_lead_map_reduce ... (0.307s) Ok
    ddl_sharding_key[vinyl].test_select_wont_lead_map_reduce ... (0.307s) Ok
```